teamsyncd/teamsyncd.cpp

```diff
@@ -34,7 +42,13 @@ int main(int argc, char **argv)
             while (true)
             {
                 Selectable *temps;
-                s.select(&temps);
+                s.select(&temps, 1000); // check every second
+
+                auto diff = duration_cast<seconds>(steady_clock::now() - start_time);
+                if(diff.count() > WR_PENDING_TIME)
```

Comment:
Like those for bgp and neighsyncd,   will making the pending time configurable be helpful?
```
root@sonic:~# config warm_restart 
Usage: config warm_restart [OPTIONS] COMMAND [ARGS]...

  warm_restart-related configuration tasks

Options:
  -s, --redis-unix-socket-path TEXT
                                  unix socket path for redis connection
  --help                          Show this message and exit.

Commands:
  bgp_timer
  disable
  enable
  neighsyncd_timer
```

Reply:
Good idea. Done

Reply:
https://github.com/Azure/sonic-utilities/pull/409